test/article/model.js

```diff
@@ -4,16 +4,17 @@
 var should = require('should'),
     app = require('../../server'),
     mongoose = require('mongoose'),
-    User = mongoose.model('User'),
-    Article = mongoose.model('Article');
+    User = mongoose.model('User');
+    //Article = mongoose.model('Article');
```

Comment:
@CodeMuhammed Please check thoroughly for any use of the Article, if its not being used please delete the commented codes. Same goes for other codes.